insuree/migrations/0025_auto_20240731_1335.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 4.2.13 on 2024-07-31 13:35
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
To make is specific to an implementation, this should be in another Comoros specific module like openimis-be-insuree_comoros_py.